### DISCUSSION TITLE
Jupyter Server

### DISCUSSION THREAD

**Discussion by AmanuelDagnew** [cooperrc/computational-mechanics#158] [Q&A]
I submitted a link to project 1 instead of a pdf and I was trying to go back and print a pdf to submit. When I click the link it says 404 not found. When I go to Jupyter and open up project 1 its empty like no work has been done. When I go to my history and click project 1 it says 404 not found. Can any one help me figure this out?

**Comment by cooperrc** [comment]
That stinks, if you did not save the work in the `work` folder it would have been deleted. Make sure as soon as you open the HW/project/readings you move the `computational-mechanics` folder to `work`.

**Reply by AmanuelDagnew** [reply]
It is okay, I had autosave on but I don't know what happened. I remember the steps so it won't take me that long. 

Thank you
